### PR TITLE
feat(constitution): complete unified execution closeout

### PR DESCRIPTION
**Pull request by jamesryancooper**
Policy reference: `.octon/framework/agency/practices/pull-request-standards.md`
Reminder: Run `@codex review`.

## What

Complete Octon's unified execution constitution clean-break closeout by
binding the final claim to current live runtime and disclosure evidence,
propagating canonical run ids through specialized workflows, retaining a
real brownfield retrofit walkthrough, and committing the proposal packet
that drove the implementation.

## Why

The repository needed an honest repo-local closeout state where the final
claim is controlled by current validators and retained current evidence
instead of historical March 29-30, 2026 certification artifacts.

No-Issue: repo-local constitutional closeout and evidence promotion.

## How

Moved the execution-governance implementation into `authority_engine`,
fixed canonical workflow discovery, propagated explicit `run_id` binding
through specialized proposal workflows, emitted current proof-plane and
RunCard artifacts for successful consequential runs, regenerated the
proposal registry from manifests, repaired archived proposal lineage that
drifted after the authority move, and retained a brownfield retrofit
walkthrough for `snake-game`.

The PR also includes the proposal packet at
`.octon/inputs/exploratory/proposals/architecture/octon-unified-execution-constitution-proposal-packet`.

## Profile Selection Receipt

- Semantic version source(s): `.octon/framework/constitution/charter.yml`, `.octon/instance/charter/workspace.md`, `.octon/instance/charter/workspace.yml`
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): one branch-local clean-break; no supported transitional coexistence; bounded supported envelope remains `MT-B / WT-2 / LT-REF / LOC-EN` on `repo-shell` + `repo-local-governed`; closeout fails closed unless validators and status matrix are green; brownfield walkthrough retained against a real sibling repo
- Tie-break status: none
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): `n/a`

## Implementation Plan

- Workstreams and scope: WS0 runtime cutover tightening, WS1 authority ownership completion, WS3/WS5 current proof and disclosure retention, WS6 brownfield walkthrough retention, proposal-packet retention
- Public interfaces/types/contracts affected: workflow CLI `run_id`/mission wording, specialized workflow run-id propagation, closure manifest, closeout status matrix, authored HarnessCard, current release HarnessCard, proposal registry projection, archived proposal provenance
- Test scenarios: authority-engine tests, kernel pipeline tests, support-target validator, Phase 4 proof/lab validator, final closeout validator, proposal-registry deterministic rebuild/check, successful current `validate-proposal` and `audit-architecture-proposal` runs
- Assumptions/defaults: supported live envelope stays bounded; browser/api packs stay governed but unadmitted outside the proved envelope; brownfield walkthrough proves retrofit executability for Octon's closeout, not downstream full adoption of the target repo

## Impact Map (code, tests, docs, contracts)

- Code: runtime authority engine, kernel pipeline/workflow run-id handling, CLI wording
- Tests: `cargo test -p octon_authority_engine`, `cargo test -p octon_kernel pipeline:: -- --nocapture`
- Docs: proposal packet, brownfield retrofit playbook, migration evidence bundle, archived proposal lineage docs
- Contracts: closure manifest, closeout reviews, status matrix, HarnessCard, release disclosure card, contract registry

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

none

## Tradeoffs

The live claim remains intentionally bounded. `WT-3`, browser/UI host
surfaces, GitHub/CI host surfaces, extended context, secondary locale,
and experimental model tiers remain reduced, staged, experimental, or
deny-by-default per the support-target matrix.

## Testing

- `cargo test -p octon_authority_engine`
- `cargo test -p octon_kernel pipeline:: -- --nocapture`
- `bash .octon/framework/assurance/runtime/_ops/scripts/generate-proposal-registry.sh --check`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-support-target-live-claims.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase4-proof-lab-enforcement.sh`
- `bash .octon/framework/assurance/governance/_ops/scripts/assert-unified-execution-closure.sh`
- `cargo run --manifest-path framework/engine/runtime/crates/Cargo.toml -p octon_kernel -- workflow run validate-proposal --run-id uec-validate-proposal-20260331-success-3 --set proposal_path=.octon/inputs/exploratory/proposals/design/studio-graph-ux-design-package`
- `cargo run --manifest-path framework/engine/runtime/crates/Cargo.toml -p octon_kernel -- workflow run audit-architecture-proposal --run-id uec-audit-architecture-20260331-2 --set proposal_path=.octon/inputs/exploratory/proposals/.archive/architecture/execution-constitution-completion-closeout`

## Rollout

Branch-local clean-break on merge. No transitional coexistence. Merge only
while the bounded validators remain green.

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [ ] Medium [x] High
- Rollback plan: revert the closeout commit and restore the prior bounded claim surfaces
- Rollback handle: `git revert d592b959e`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes: no OpenAPI changes; runtime/disclosure contract surfaces updated in repo-local YAML/JSON schema families
- Threat model update/link: repo-local governance and authority/disclosure routing updated; no separate threat model doc added

## Observability and Performance

- Traces/logs/metrics for changed flows: canonical run receipts, proof-plane reports, measurement summaries, intervention logs, replay manifests, validator logs in retained evidence roots
- Representative traces for high risk changes: `uec-validate-proposal-20260331-success-3`, `uec-audit-architecture-20260331-2`
- Performance or bundle impact: negligible runtime overhead; larger retained evidence footprint by design

## License and Provenance

- New dependencies and licenses: none
- Generated code/templates provenance notes: proposal registry regenerated from committed proposal manifests; workflow evidence produced by current runtime and validators

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [x] All review conversations resolved

## Screenshots / Notes

No visual changes. Proposal packet included in this PR as requested.

### CODE REVIEW

File: .octon/framework/engine/runtime/crates/kernel/src/workflow.rs [hunk 1]
```diff
@@ -724,11 +730,15 @@ pub fn run_create_design_package_from_octon_dir(
     let design_proposals_root = repo_root.join(DESIGN_PACKAGES_ROOT_REL);
     let reports_root = repo_root.join(REPORTS_ROOT_REL);
     let workflow_bundles_root = repo_root.join(WORKFLOW_REPORTS_ROOT_REL);
+    let workflow_request_id = options
+        .run_id
+        .clone()
+        .unwrap_or_else(|| format!("create-design-proposal-{}", options.package_id));
     let workflow_auth = authorize_execution(
         &runtime_cfg,
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/engine/runtime/crates/kernel/src/workflow.rs]: Fixed in `cff649469`.

Specialized proposal workflows now route caller-provided `--run-id` values through canonical validation and reuse checks before authorization, and the kernel has regression coverage for both invalid path-separator ids and reused ids:
- `cargo test -p octon_kernel validate_proposal_rejects_ -- --test-threads=1`

The specialized entrypoints no longer thread raw caller ids directly into canonical run roots.

File: .octon/framework/engine/runtime/crates/authority_engine/src/implementation.rs [hunk 2]
```diff
@@ -4925,10 +4954,365 @@ pub fn finalize_execution(
...
+            json!({"metric_id":"receipt-count","label":"Retained lifecycle receipts","value":1,"unit":"count"}),
+            json!({"metric_id":"checkpoint-count","label":"Retained checkpoints","value":3,"unit":"count"}),
+            json!({"metric_id":"proof-plane-count","label":"Run-local proof-plane reports","value":7,"unit":"count"}),
+            json!({"metric_id":"intervention-count","label":"Material interventions","value":0,"unit":"count"}),
+        ]
+    } else {
+        vec![
+            json!({"metric_id":"validator-count","label":"Proposal validators executed","value":2,"unit":"count"}),
+            json!({"metric_id":"proof-plane-count","label":"Explicit proof-plane reports emitted by the workflow","value":3,"unit":"count"}),
+            json!({"metric_id":"intervention-count","label":"Material interventions","value":0,"unit":"count"}),
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/engine/runtime/crates/authority_engine/src/implementation.rs:5200]: Fixed in `cff649469`.

`materialize_run_disclosure` now keeps failed-run measurements workflow-agnostic by deriving generic receipt/checkpoint/proof-plane counts instead of proposal-validator-specific metrics, and the new regression test covers a failed `archive_proposal` path:
- `cargo test -p octon_authority_engine failed_run_measurement_artifacts_remain_workflow_agnostic -- --test-threads=1`

The non-success summary is now generic as well, so failed create/promote/archive-style runs no longer emit misleading validate-proposal evidence.
